capstone-app/src/app/modules/history/wiki-media/wiki-media.component.html

```diff
@@ -4,9 +4,10 @@
       <mat-card-title class="history-title">
         {{title}}
       </mat-card-title>
-      <div class="text">
-      {{body}}
+      <div class="text" *ngFor="let paragraph of body">
+        <p>{{paragraph}}</p>
       </div>
+      <i>All the data above was obtained <a href="{{UrlForAttribution}}" target="_blank">here</a>.</i>
```

Comment:
Make sure to change the color of here to blue and to underline it. I was having a hard time figuring out where to click.

Reply:
Will do, thanks!